src/HyperFragmentView.tsx

```diff
@@ -24,15 +24,31 @@ var HyperFragmentViewManager: any;
 
 if (Platform.OS === 'android') {
   HyperFragmentViewManager = requireNativeComponent('HyperFragmentViewManager');
+} else {
+  HyperFragmentViewManager = requireNativeComponent('HyperFragmentViewManagerIOS');
 }
 
 const createFragment = (viewId: number, namespace: string, payload: string) => {
-  UIManager.dispatchViewManagerCommand(
-    viewId,
-    //@ts-ignore
-    UIManager.HyperFragmentViewManager.Commands.process.toString(),
-    [viewId, namespace, payload]
-  );
+  if (Platform.OS == 'android') {
+    UIManager.dispatchViewManagerCommand(
+      viewId,
+      //@ts-ignore
+      UIManager.HyperFragmentViewManager.Commands.process.toString(),
+      [viewId, namespace, payload]
+    );
+  }
+  else {
+    const commandId = UIManager.getViewManagerConfig(
```

Comment:
why is the iOS specific code different?

Reply:
`UIManager.HyperFragmentViewManagerIOS.Commands.process` is giving undefined in case of IOS.